config/defaults.js

```diff
@@ -41,6 +41,13 @@ module.exports = {
     randomizationFactor: 0.25
   },
 
+  features: {
```

Comment:
note here: I would like to move towards grouping feature related config together.  in a future PR if this method is acceptable, I can move the images we store in the config under features:<name>:image:<image>